### ISSUE TITLE
Invalid .js file extension for ESM exports

### ISSUE THREAD

**Issue by jaydenseric** [thekelvinliu/country-code-emoji#7]
https://github.com/thekelvinliu/country-code-emoji/blob/7eef3bcac4d48731e6bef9147d07a587f9ae4b0f/package.json#L12

This is incorrect, and result in a Node.js runtime error:

```js
import countryCodeEmoji from 'country-code-emoji';
```

```
(node:26808) Warning: To load an ES module, set "type": "module" in the package.json or use the .mjs extension.
/[redacted]/node_modules/country-code-emoji/src/index.js:15
export function countryCodeEmoji(cc) {
^^^^^^
SyntaxError: Unexpected token 'export'
    at wrapSafe (internal/modules/cjs/loader.js:915:16)
    at Module._compile (internal/modules/cjs/loader.js:963:27)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1027:10)
    at Module.load (internal/modules/cjs/loader.js:863:32)
    at Function.Module._load (internal/modules/cjs/loader.js:708:14)
    at ModuleWrap.<anonymous> (internal/modules/esm/translators.js:195:29)
    at ModuleJob.run (internal/modules/esm/module_job.js:145:37)
    at async Loader.import (internal/modules/esm/loader.js:182:24)
    at async Object.loadESM (internal/process/esm_loader.js:68:5)
```

**Comment by jaydenseric** [comment]
Here is the really gross workaround for people totally blocked from using `import` until this bug is fixed:

```js
import { createRequire } from 'module';

const require = createRequire(import.meta.url);

const { countryCodeEmoji } = require('country-code-emoji');
```

See:

https://nodejs.org/api/module.html#module_module_createrequire_filename

**Comment by thekelvinliu** [comment]
hey @jaydenseric thanks for bringing this to my attention! this is fixed in v2.3.0.
